src/utils/base-route.ts

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-console */
+
+/**
+ * stores BUSINESS_ID, TEMP_REG_NUMBER, BASE_URL, VUE_ROUTER_BASE from initial route.
+ */
+
+export function setBaseRouteAndBusinessId (pathname: string, processEnvBaseUrl: string, rootUrl: string): Promise<any> {
+  if (!processEnvBaseUrl || !pathname || !rootUrl) {
+    return Promise.reject(new Error('Missing environment variables'))
```

Comment:
As discussed, this _should_ work without the Promise.reject call. You should be able to simply throw a new error to reject this "wrapped" promise.

Actually, I don't see anything async in here -- you can simplify it!

Reply:
Good point, I'll change the return type to 'void' and remove the Promise rejects